server/routes/aws/cloudwatch_tasks.py

```
@@ -0,0 +1,423 @@
+        return json.dumps(data, ensure_ascii=False)
+    except Exception:
+        return str(data)
+
+
+@celery_app.task(
+    bind=True, max_retries=3, default_retry_delay=30, name="cloudwatch.process_alarm"
+)
+def process_cloudwatch_alarm(
+    self,
+    payload: Dict[str, Any],
+    metadata: Optional[Dict[str, Any]] = None,
```

Ruff (0.15.14)

[warning] Unused function argument: `metadata`

(ARG001)